src/statistical/unigram.jl

```diff
@@ -0,0 +1,177 @@
+"""
+struct Sentencepiecemodel
+  vocab::Array{String,1}
+  logprob::Array{Float64,1}
+end
+structure, To hold vocabulary,log probability and index  
+"""
+struct Sentencepiecemodel
+  vocab::Array{String,1}
+  logprob::Array{Float64,1}
+end
+
+function load(ty::Type{T}, name::String) where T<:PretrainedTokenizer
+    filepath = @datadep_str name
+    filepath = "$filepath/$name"
+    load(filepath)  
+end
+    
+function load(path)
+    vocab = readlines(path)
+    vocabnew = split.(vocab, "\t")
+    vo = []
+    for i in 1:length(vocab)
+        vocab1 = vocabnew[i][1]
+        vocab1 = replace(vocab1, "▁"=>"_")
+        push!(vo, vocab1)
+    end
+    vocab1 = convert(Array{String,1},vo)
+    logprob = []
+    for i in 1:length(vocab)
+        logp = vocabnew[i][2]
+        push!(logprob, logp)    
+    end
+    logp = convert(Array{String,1}, logprob)
+    logp =  parse.(Float64, logprob)
+    spm = Sentencepiecemodel(vocab1, logp)
+    return spm
+end
+
+# to get index of given string
+function getindex(sp::Sentencepiecemodel, text)
+    id_list = findall(x->x==text, sp.vocab)
+    length(id_list) == 0 && return 2 #unk token index 
```

Comment:
Can we have a field in sp for holding the unknown token index?
And then replace 2 by that index. For Albert, we can set 2 by default.

Reply:
yeah you are right that will be much better

Reply:
I think instead of setting unk_id to 2 
we can search for "<unk>"  which is the default for all other models